packages/kamino-sdk/src/Kamino.ts

```diff
@@ -3870,6 +3883,64 @@ export class Kamino {
     return this.getPriceRangePercentageBasedFromPrice(poolPrice, lowerPriceBpsDifference, upperPriceBpsDifference);
   }
 
+  /**
+   * Get the raw rebalancing params given the strategy type
+   */
+  async readRebalancingParamsFromChain(strategy: PublicKey | StrategyWithAddress): Promise<RebalanceFieldsDict> {
+    const strategyWithAddress = await this.getStrategyStateIfNotFetched(strategy);
+    let rebalanceKind = numberToRebalanceType(strategyWithAddress.strategy.rebalanceType);
+
+    if (rebalanceKind.kind === Manual.kind) {
```

Comment:
you still need upper range and lower range here (and everywhere if not included)